### ISSUE TITLE
About the quality of download Youtube video

### ISSUE THREAD

**Issue by xwsgithub** [ParitoshParmar/MTL-AQA#2]
What quality video should I download from Youtube?
Thank you~

**Comment by ParitoshParmar** [comment]
@xwsgithub That really depends on what resolution of videos you intend to use and how much HDD space you want to dedicate. I personally had downloaded them at 640*320 pixels. In this repo, we downsize them to 171*128 pixel videos; although you can use higher resolution videos. Cheers!

**Comment by xwsgithub** [comment]
I am very grateful for your reply. I used a 640320 pixel video. After I ran for 100 epochs, I found that the highest correlation was 88.2, which did not reach 90.44. Is there any other reason that affects this accuracy? Sorry to bother you.

**Comment by ParitoshParmar** [comment]
@xwsgithub 88.20% correlation is actually less than even our STL. My guess is that reason might be the difference in our PyTorch versions? As far as I can remember, I had used 1.0.0. You might want to give that version a try. Let me know how it goes.

**Comment by xwsgithub** [comment]
My PyTorch versions is 1.4.0 now.I will try it with 1.0.0 and tell you the result.Thank you!

**Comment by xwsgithub** [comment]
Sorry to bother you, the latest result is 88.1~And my python version is 3.7.9.

**Comment by ParitoshParmar** [comment]
@xwsgithub Not sure exactly, but following three possibilities come to mind:
1. Are you initializing with Sports-1M pretrained C3D backbone?
2. Have you correctly extracted the frames? I have included the script that I used to extract the frames.
3. The performance you reported is similar to what we got on UNLV-Dive (Table 5 bottom). Are you using UNLV-Dive or MTL-AQA dataset?

**Comment by xwsgithub** [comment]
1. Sports-1M pretrained C3D weights is downloaded from http://imagelab.ing.unimore.it/files/c3d_pytorch/c3d.pickle.
2. I used the script you gave to extract the frames.
3. I am sure I used the MTL-AQA dataset.

I download the code from the github, and only some of the parameters for C3D-AVG in the file were modified, and the others remained unchanged.

**Comment by ParitoshParmar** [comment]
@xwsgithub Not sure what's causing the problem, you seem to be doing correctly (or at least, nothing that I can see as a mistake). Me and other groups have been able to reproduce results. When you say `only some of the parameters for C3D-AVG in the file were modified` -- what parameters are you changing?

**Comment by xwsgithub** [comment]
Like input data dims,input_resize and caption_lstm_dim_vid.
I'm very sorry for the trouble to you and the team!

**Comment by ParitoshParmar** [comment]
@xwsgithub Those are critical parameters that can affect the results. So, difference in results is expected with changes in those parameters. Any particular reason you are changing those parameters?

**Comment by xwsgithub** [comment]
I mean i change the default MSCADC configs to the C3DAVG configs~

**Comment by ParitoshParmar** [comment]
@xwsgithub Sorry, in that case, I am not sure what's causing the problem.

**Comment by zahidzqj** [comment]
> 1. Sports-1M pretrained C3D weights is downloaded from http://imagelab.ing.unimore.it/files/c3d_pytorch/c3d.pickle.
> 2. I used the script you gave to extract the frames.
> 3. I am sure I used the MTL-AQA dataset.
> 
> I download the code from the github, and only some of the parameters for C3D-AVG in the file were modified, and the others remained unchanged.
where  is  the script  to extract the frames？I can't find it in the code.

**Comment by xwsgithub** [comment]
> > 1. Sports-1M pretrained C3D weights is downloaded from http://imagelab.ing.unimore.it/files/c3d_pytorch/c3d.pickle.
> > 2. I used the script you gave to extract the frames.
> > 3. I am sure I used the MTL-AQA dataset.
> > 
> > I download the code from the github, and only some of the parameters for C3D-AVG in the file were modified, and the others remained unchanged.
> > where  is  the script  to extract the frames？I can't find it in the code.

/MTL-AQA_dataset_release/frame_extractor.sh

**Comment by xwsgithub** [comment]
> @xwsgithub Sorry, in that case, I am not sure what's causing the problem.

You are too kind! Thank you very much for your answers during this time!

**Comment by zahidzqj** [comment]
Thank you very much,  I solved this problem. I ran C3D-LSTM--PyTorch-master/train_c3d_lstm.py，and got the following result：
rue scores:  [5.962239152037099, 5.8069725074528, 6.707519046041736, ................................]
Correlation:  0.8506981639585309

Is that right？

**Comment by xwsgithub** [comment]
Sorry, I just run the C3D-AVG~

**Comment by zahidzqj** [comment]
The AQA folder is as follows：
![image](https://user-images.githubusercontent.com/37623070/117390814-69476400-af21-11eb-989e-e8a352c31f87.png)

when I run train_test_C3DAVG.py，a  error occurred:No such file or directory:**/frames/02/*.jpg 
so where is the file( 01 02 03 04 05 06 07 09 10 13 14 17 18 21 22 26) in frame_extractor.sh?

**Comment by ParitoshParmar** [comment]
@xwsgithub Thank you!
@zahidzqj This repo is not for C3D-LSTM/AQA-7. Please open this issue in C3D-LSTM repo.
